### PR TITLE
Fixes a null override

### PR DESCRIPTION
**Pull request by ike709**
See https://github.com/ParadiseSS13/Paradise/pull/17581

### CODE REVIEW

**Comment by adamsong** [comment]
Does this not cause all the references to `AI` in the code point to the wrong variable?

File: code/game/mecha/mecha_wreckage.dm [hunk 1]
```diff
@@ -80,7 +80,7 @@
 			to_chat(user, span_warning("You don't see anything that can be pried with [I]!"))
 
 
-/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, null, obj/item/aicard/card)
+/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, mob/living/silicon/ai/AI, obj/item/aicard/card)
```
Comment (AffectedArc07) [inline code/game/mecha/mecha_wreckage.dm]:
```suggestion
/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, mob/living/silicon/ai/the_ai, obj/item/aicard/card)
```
`AI` is a local var on this type itself and I am worried about fuckups with this.

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/game/mecha/mecha_wreckage.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/mecha/mecha_wreckage.dm]: Wait this isnt my fucking codebase

**Comment by ike709** [comment]
The fact that `AI` is both the name of a positional arg in the proc def and an arg in the parent proc def pains me.

**Comment by ToasterBiome** [comment]
Duplicate of https://github.com/yogstation13/Yogstation/pull/13581

Will merge whichever has a less stupid name for the variable soonest

**Pull request closed without merging**